src/api/java/mekanism/api/math/FloatingLong.java

```diff
@@ -482,7 +555,33 @@ public FloatingLong divide(FloatingLong toDivide) {
      * @throws ArithmeticException if {@code toDivide} is zero.
      */
     public FloatingLong divide(long toDivide) {
```

Comment:
This method should instead do a similar thing to the divide(FloatingLong) (in that it does the copy) and have the actual logic added below be done in the divideEquals that takes a long as a parameter.

Reply:
Done, but could use a review to make sure.